packages/grafana-e2e-selectors/src/selectors/components.ts

```diff
@@ -354,6 +359,9 @@ export const versionedComponents = {
       content: {
         '11.1.0': 'data-testid panel content',
       },
+      headerContainer: {
+        '9.5.0': 'data-testid header-container',
```

Comment:
@kaydelaney Plugins don't have any way to import these constants, so every time a testId changes all of our tests that use these selectors break, and it also makes running against a matrix of Grafana versions very frustrating. 

I imagine this change, while well intentioned, will cause a cascade of failing e2e tests in plugins when folks start running against 12.1

Reply:
oh wait nvm, there is a `@grafana/e2e-selectors` package, I was looking for `@grafana/grafana-e2e-selectors`, I guess we should be using that 🤦 .

But still if we can avoid changing these as much as possible that would be great, as this selector wasn't in the package before 12.1.

Reply:
for reference, it was `header-container` in <12.1 and in 12.1 it is now `data-testid header-container`, so all selectors used in plugins before 12.1 will break, and plugins running an e2e matrix will need to use a regex to support <12.1 & 12.1